### PR TITLE
Text adjustments legal texts (EXPOSUREAPP-7665)

### PR DESCRIPTION
**Pull request by NataliaLemmerth**

### CODE REVIEW

**Review by ChristinaTsigganos — APPROVED** [review]
approved

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings [hunk 1]
```diff
@@ -208,7 +208,7 @@
 
 "ExposureSubmission_TestCertificate_Information_Legal_Headline" = "Datenschutz und Datensicherheit";
 
-"ExposureSubmission_TestCertificate_Information_Legal_Text_1" = "Die Anforderung und Verwendung des digitalen COVID-Testzertifikats ist freiwillig. Der Nachweis eines negativen Testzertifikats kann auch auf andere Weise erbracht werden.";
+"ExposureSubmission_TestCertificate_Information_Legal_Text_1" = "Die Anforderung und Verwendung des digitalen COVID-Testzertifikats ist freiwillig. Der Nachweis eines negativen Testzergebnisses kann auch auf andere Weise erbracht werden.";
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings:211]:
```suggestion
"ExposureSubmission_TestCertificate_Information_Legal_Text_1" = "Die Anforderung und Verwendung des digitalen COVID-Testzertifikats ist freiwillig. Der Nachweis eines negativen Testergebnisses kann auch auf andere Weise erbracht werden.";
```

**Review by pascal-brause — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings (hunk 1 above)
Comment (pascal-brause) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings:211]: I think there is a "z" too much.

Comment (NataliaLemmerth) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings]: Definitely!

**Review by marcussc — APPROVED** [review]

**Review by pascal-brause — APPROVED** [review]

**Pull request merged**